pages/projects/[projectId]/versions/[versionId]/index.tsx

```diff
@@ -219,6 +219,13 @@ export default function VersionPage({
     [version]
   );
 
+  const positiveChartPoints = data.filter((point) => point.remaining >= 0);
```

Comment:
Can we keep track oh how many below 0 we have, so that they can still display on the graph as numbers?

Reply:
I'll do this in the follow up ticket with the warning message!